classes/blocks/class-socialmedia.php

```diff
@@ -95,17 +79,22 @@ public function __construct() {
 						'type'    => 'string',
 						'default' => '',
 					],
+					'embed_code'        => [
+						'type'    => 'string',
+						'default' => '',
+					],
 				],
 			]
 		);
+
+		add_action( 'enqueue_block_editor_assets', [ self::class, 'enqueue_editor_assets' ] );
+		add_action( 'wp_enqueue_scripts', [ self::class, 'enqueue_frontend_assets' ] );
 	}
 
 	/**
-	 * Get all the data that will be needed to render the block correctly.
-	 *
-	 * @param array $fields This is the array of fields of this block.
+	 * Required by the `Base_Block` class.
 	 *
-	 * @return array The data to be passed in the View.
+	 * @param array $fields Unused, required by the abstract function.
```

Comment:
`$fields` is actually used, maybe you meant something else that isn't used? Also the return parameter should still be in the doc comment I think, not sure why it doesn't make the lint job fail :confused: